docs/getting_started/tutorial.md

```diff
@@ -705,108 +705,130 @@ This method is now available from our templates. Update `blog_index_page.html` t
 
 (tutorial_categories)=
 
-### Categories
+### Authors
+
+You probably want your blog posts to have authors, which is an essential feature of blogs. The way to go about this is to have a fixed list, managed by the site owner through a separate area of the [admin interface](https://guide.wagtail.org/en-latest/concepts/wagtail-interfaces/#admin-interface).
 
-Let's add a category system to our blog. Unlike tags, where a page author can bring a tag into existence simply by using it on a page, our categories will be a fixed list, managed by the site owner through a separate area of the admin interface.
+First, define an `Author` model. This model isn't a page in its own right. You have to define it as a standard Django `models.Model` rather than inheriting from `Page`. Wagtail introduces the concept of **Snippets** for reusable pieces of content, but they don't exist as part of the page tree themselves. You can manage **Snippets** through the [admin interface](https://guide.wagtail.org/en-latest/concepts/wagtail-interfaces/#admin-interface). You can register a model as a **Snippet** by adding the `@register_snippet` decorator. Also, you can use all the fields types that you've used so far on pages on snippets too. 
```

Comment:
I’d like to hear your thoughts on this but I have a feeling that "snippet" is generic enough of a term it shouldn’t be emphasized as **Snippet** every time we use it. Them not existing as part of the page tree is what makes them useful.

```suggestion
First, define an `Author` model. This model isn't a page in its own right. You have to define it as a standard Django `models.Model` rather than inheriting from `Page`. Wagtail introduces the concept of **Snippets** for reusable pieces of content which don't exist as part of the page tree themselves. You can manage snippets through the [admin interface](https://guide.wagtail.org/en-latest/concepts/wagtail-interfaces/#admin-interface). You can register a model as a snippet by adding the `@register_snippet` decorator. Also, you can use all the fields types that you've used so far on pages on snippets too. 
```

Reply:
I bold Snippets because it's a term/concept particular to Wagtail. Perhaps we should treat it like we would any other term specific to Wagtail., for instance, the Wagtail Sidebar. This means adopting the official capitalization and providing a link explaining it.